src/main/java/com/endava/Ticket_Management_System/model/Event.java

```diff
@@ -0,0 +1,107 @@
+package com.endava.Ticket_Management_System.model;
+import jakarta.persistence.*;
+import java.io.Serializable;
+import java.util.Date;
+
+
+@Entity
+@Table(name = "Event")
+public class Event implements Serializable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int eventID;
+
+    @ManyToOne
+    @JoinColumn(name = "venueID")
+    private Venue venueID;
+
+    @ManyToOne
+    @JoinColumn(name = "eventTypeID")
+    private EventType eventTypeID;
+
+//    @OneToMany(mappedBy = "Event",cascade = CascadeType.ALL, fetch = FetchType.LAZY)
+//    private List<TicketCategory> ticketCategoryList;
+
+    @Column(name = "eventDescription")
+    private String eventDescription;
+
+    @Column(name = "eventName")
+    private String eventName;
+
+    @Column(name = "startDate")
+    private Date startDate;
+
+    @Column(name = "endDate")
+    private Date endDate;
+
+    public Event(){
```

Comment:
Why didn't you use lombok for constructors, getter and setters? I see you have it as a dependency